resources/eventing/charts/nats/templates/secret.yaml

```diff
@@ -0,0 +1,24 @@
+{{- $secret := (lookup "v1" "Secret" .Release.Namespace "eventing-nats-accounts") -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: eventing-nats-accounts
```

Comment:
(nit) instead of hard coding name, can we follow the same approach as in [configmap.yaml](https://github.com/kyma-project/kyma/blob/f41c24f41f686ba420ea063d93f654e915e8cc1a/resources/eventing/charts/nats/templates/configmap.yaml)?